Type 'Music' is missing the following properties

Hi,

There seems to be an issue with the types since 2.10, which seems linked to https://github.com/elevenlabs/elevenlabs-js/pull/289.

When running `tsc` on my code using 2.10, everything works 

With 2.17 I have the following error:

```
../../common/temp/node_modules/.pnpm/@elevenlabs+elevenlabs-js@2.17.0/node_modules/@elevenlabs/elevenlabs-js/wrapper/ElevenLabsClient.d.ts:19:9 - error TS2416: Property 'music' in type 'ElevenLabsClient' is not assignable to the same property in base type 'ElevenLabsClient'.
  Type 'import("/Users/melvin/Dev/red/api/common/temp/node_modules/.pnpm/@elevenlabs+elevenlabs-js@2.17.0/node_modules/@elevenlabs/elevenlabs-js/wrapper/music").Music' is not assignable to type 'import("/Users/melvin/Dev/red/api/common/temp/node_modules/.pnpm/@elevenlabs+elevenlabs-js@2.17.0/node_modules/@elevenlabs/elevenlabs-js/api/resources/music/client/Client").Music'.
    The types returned by 'composeDetailed(...)' are incompatible between these types.
      Type 'Promise<MultipartResponse>' is missing the following properties from type 'HttpResponsePromise<ReadableStream<Uint8Array<ArrayBufferLike>>>': innerPromise, unwrappedPromise, unwrap, withRawResponse

19     get music(): Music;
           ~~~~~

../../common/temp/node_modules/.pnpm/@elevenlabs+elevenlabs-js@2.17.0/node_modules/@elevenlabs/elevenlabs-js/wrapper/music.d.ts:32:5 - error TS2416: Property 'composeDetailed' in type 'Music' is not assignable to the same property in base type 'Music'.
  Type '(request?: BodyComposeMusicWithADetailedResponseV1MusicDetailedPost | undefined, requestOptions?: RequestOptions | undefined) => Promise<...>' is not assignable to type '(request?: BodyComposeMusicWithADetailedResponseV1MusicDetailedPost | undefined, requestOptions?: RequestOptions | undefined) => HttpResponsePromise<...>'.
    Type 'Promise<MultipartResponse>' is missing the following properties from type 'HttpResponsePromise<ReadableStream<Uint8Array<ArrayBufferLike>>>': innerPromise, unwrappedPromise, unwrap, withRawResponse

32     composeDetailed(request?: ElevenLabs.BodyComposeMusicWithADetailedResponseV1MusicDetailedPost, requestOptions?: Music.RequestOptions): Promise<MultipartResponse>;
       ~~~~~~~~~~~~~~~


Found 2 errors in 2 files.

Errors  Files
     1  ../../common/temp/node_modules/.pnpm/@elevenlabs+elevenlabs-js@2.17.0/node_modules/@elevenlabs/elevenlabs-js/wrapper/ElevenLabsClient.d.ts:19
     1  ../../common/temp/node_modules/.pnpm/@elevenlabs+elevenlabs-js@2.17.0/node_modules/@elevenlabs/elevenlabs-js/wrapper/music.d.ts:32
```

With 2.22 (current latest) I have the following error:

```
@elevenlabs/elevenlabs-js/wrapper/ElevenLabsClient.d.ts:21:9 - error TS2416: Property 'music' in type 'ElevenLabsClient' is not assignable to the same property in base type 'ElevenLabsClient'.
  Type 'Music' is missing the following properties from type 'Music': _options, _compositionPlan, __compose, __composeDetailed, and 3 more.

21     get music(): Music;
           ~~~~~


Found 1 error in ../../common/temp/node_modules/.pnpm/@elevenlabs+elevenlabs-js@2.22.0/node_modules/@elevenlabs/elevenlabs-js/wrapper/ElevenLabsClient.d.ts:21
```

My code doesn't do anything else but calling `studio.projects.create` on an `ElevenLabsClient` instance.

--- a/src/wrapper/ElevenLabsClient.ts
+++ b/src/wrapper/ElevenLabsClient.ts
@@ -4,6 +4,7 @@ import type * as core from "../core";
 import * as errors from "../errors";
 import { WebhooksClient } from "./webhooks";
 import { Music } from "./music";
+import { Music as GeneratedMusic } from "../api/resources/music/client/Client";
 import { SpeechToText } from "./speechToText";
 
 export declare namespace ElevenLabsClient {
@@ -39,12 +40,13 @@ export class ElevenLabsClient extends FernClient {
         return this._customWebhooks;
     }
 
-    // @ts-expect-error - Intentionally overriding with wrapper Music that has enhanced composeDetailed
-    public override get music(): Music {
+    public override get music(): GeneratedMusic {
         if (!this._customMusic) {
             this._customMusic = new Music(this._options);
         }
-        return this._customMusic as any;
+        // Return wrapper Music cast as GeneratedMusic to maintain type compatibility
+        // The wrapper has enhanced composeDetailed that returns MultipartResponse
+        return this._customMusic as any as GeneratedMusic;
     }
 
     public get speechToText(): SpeechToText {

--- a/src/wrapper/music.ts
+++ b/src/wrapper/music.ts
@@ -28,8 +28,8 @@ export interface MultipartResponse {
 
 export class Music {
     private _client: GeneratedMusic;
-    protected readonly _options: Music.Options;
-    protected _compositionPlan: CompositionPlan | undefined;
+    private readonly _options: Music.Options;
+    private _compositionPlan: CompositionPlan | undefined;
 
     constructor(options: Music.Options = {}) {
         this._options = options;

